README.md

```diff
@@ -1,14 +1,15 @@
+
+# Optimized Einsum
+
 [![Build Status](https://travis-ci.org/dgasmith/opt_einsum.svg?branch=master)](https://travis-ci.org/dgasmith/opt_einsum)
 [![codecov](https://codecov.io/gh/dgasmith/opt_einsum/branch/master/graph/badge.svg)](https://codecov.io/gh/dgasmith/opt_einsum)
 [![Anaconda-Server Badge](https://anaconda.org/conda-forge/opt_einsum/badges/version.svg)](https://anaconda.org/conda-forge/opt_einsum)
 [![PyPI](https://img.shields.io/pypi/v/opt_einsum.svg)](https://pypi.org/project/opt-einsum/#description)
 [![PyPIStats](https://img.shields.io/pypi/dm/opt_einsum)](https://pypistats.org/packages/opt-einsum)
 [![Documentation Status](https://readthedocs.org/projects/optimized-einsum/badge/?version=latest)](http://optimized-einsum.readthedocs.io/en/latest/?badge=latest)
-[![DOI](http://joss.theoj.org/papers/10.21105/joss.00753/status.svg)](https://doi.org/10.21105/joss.00753)
+[![DOI](https://joss.theoj.org/papers/10.21105/joss.00753/status.svg)](https://doi.org/10.21105/joss.00753)
 
-
-Optimized Einsum: A tensor contraction order optimizer
-======================================================
+## Optimized Einsum: A tensor contraction order optimizer
```

Comment:
I'm not sure I like this switch, badges at the top feel more comparable to me. Is there a reason for it?

Reply:
Following guideline https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041 however I will switch back if it doesn't make sense here.

Reply:
Interesting I wasn't aware of the markdown resource. I'll link it to several other communities for review.